crates/solvers/src/boundary/legacy.rs

```diff
@@ -578,6 +581,22 @@ fn to_boundary_auction_result(notification: &notification::Notification) -> (i64
 
     let auction_result = match &notification.kind {
         Kind::EmptySolution => AuctionResult::Rejected(SolverRejectionReason::NoUserOrders),
+        Kind::SimulationFailed(tx) => AuctionResult::Rejected(
+            SolverRejectionReason::SimulationFailure(TransactionWithError {
+                error: "".to_string(),
+                transaction: SimulatedTransaction {
+                    from: tx.from.into(),
+                    to: tx.to.into(),
+                    data: tx.input.clone().into(),
+                    internalization: InternalizationStrategy::SkipInternalizableInteraction,
+                    block_number: Default::default(),
```

Comment:
Still needs to be populated.

Reply:
My understanding from the description is that this is intentionally left unpopulated, same as the one below.
Still, I'd like to see something like `block_number: PLACEHOLDER_UNAVAILABLE_BLOCK` where `PLACEHOLDER_UNAVAILABLE_BLOCK = 0` that clarifies this is intentionally left blank.

Reply:
@fedgiac makes sense, added also the reference to the tracked issue https://github.com/cowprotocol/services/issues/2018